docs/troubleshooting.mdx

```diff
@@ -38,6 +38,7 @@ You can set the correct permissions by running `chmod -R 755 /var/www/pelican/st
 
 * `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
 * `502 Bad Gateway Error` or some code in plain text you either didn't install php-fpm or set the wrong version in your webserver vhost.
+* `sh: 1: mysql: not found` if you are using Mariadb ^11.4 you'll have to restore the mysql alias dy doing `cd /usr/bin;ln -s mariadb mysql` (You need mariadb-client installed on the panel machine eventhough your mariadb-server is hosted elsewhere)
```

Comment:
```suggestion
* `sh: 1: mysql: not found`: If you are using MariaDB 11.4 or higher you'll have to restore the mysql alias by running `ln -s /usr/bin/mariadb /usr/bin/mysql`.
```

Reply:
I used cd because mariadb did it using relative path

Reply:
There is no reason to use relative paths. With absolute paths it's simpler.